app/src/main/java/com/example/lexis/fragments/ProfileSettingsFragment.java

```diff
@@ -0,0 +1,154 @@
+package com.example.lexis.fragments;
+
+import android.content.Context;
+import android.os.Bundle;
+
+import androidx.annotation.NonNull;
+import androidx.annotation.Nullable;
+import androidx.fragment.app.Fragment;
+import androidx.fragment.app.FragmentManager;
+
+import android.util.Log;
+import android.view.LayoutInflater;
+import android.view.Menu;
+import android.view.MenuItem;
+import android.view.View;
+import android.view.ViewGroup;
+import android.view.inputmethod.InputMethodManager;
+import android.widget.Toast;
+
+import com.example.lexis.R;
+import com.example.lexis.databinding.FragmentProfileSettingsBinding;
+import com.example.lexis.utilities.Utils;
+import com.google.android.material.navigation.NavigationView;
+import com.parse.LogInCallback;
+import com.parse.ParseException;
+import com.parse.ParseUser;
+
+import org.parceler.Parcels;
+
+import java.util.Arrays;
+import java.util.List;
+
+public class ProfileSettingsFragment extends Fragment {
+
+    private static final String ARG_USER = "user";
+    private static final List<String> languageCodes = Arrays.asList("fr", "es", "de", "tr");
```

Comment:
I think this also shows up in the signup activity: would it be useful to pull this out to a constant file somewhere?

Reply:
That way, if you want to change the languages you support, you don't have to change multiple callsites.

Reply:
Yes, that makes sense — fixed!